### PR TITLE
chore: Enable spellchecking on all files

### PR DESCRIPTION
**Pull request by nschonni**
- Remove words that are now part of the base dictionary
- Expand the check to cover all files
- Ignore the deleted archive file and third party files
- Renamed some shortened JS variables so they didn't need to be ignored
- Update CI path filters to match new ignored files

### CODE REVIEW

**Comment by nschonni** [comment]
@mcking65 @spectranaut @jongund I have a feeling this might also cause conflicts with some open PRs, so let me know if you want me to break this up into smaller bits

File: examples/combobox/js/combobox-autocomplete.js
Comment (nschonni) [inline examples/combobox/js/combobox-autocomplete.js:257]: This appears to actually be  bug

Comment (jongund) [reply examples/combobox/js/combobox-autocomplete.js:257]: @nschonni 
Yes it is, so you can fix it.
I need to look at the combobox code again, I think it needs to be updated, there are better was to do what this code is trying to accomplish.

**Review by spectranaut — APPROVED** [review]
This is awesome -- but yes it will definitely cause some conflicts. Not sure the easiest way to break it up though?

**Comment by jongund** [comment]
@spectranaut 
@nschonni 
Can I make a branch to update the combobox code, or should I wait until this merge is finished?

**Comment by nschonni** [comment]
@jongund don't let this stop you working on anything. Feel free to apply any changes in your branch and I can always rebase this one if yours lands first

File: cspell.json
Comment (mcking65) [inline cspell.json:34]: "Checkboxes" is a valid word. Is included it unnecessary now?

Comment (nschonni) [reply cspell.json:34]: Yeah, I went through the words and confirmed the ones that now exist in the regular dictionaries now

**Comment by mcking65** [comment]
@nschonni, thank you for all your work on this.

This is kind of far reaching, but .... here we go. @jongund, I see you are aware. I hope it doesn't cause too much pain. In the end, it'll be worth it.

**Pull request merged**
